File: testprojects/c#/benchmark/domain/foursquarealternative/kilroy/Map.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.domain.foursquarealternative.whrrl;

namespace CSharpBenchmark.domain.foursquarealternative.kilroy
{
    //Functional requirement 3.2
    //Test case 150: Class domain.foursquarealternative.kilroy.Map is not allowed to have a dependency with classes from domain.foursquarealternative.glympse
    //Result: TRUE
    public class Map
    {
        public Map()
        {
            Console.WriteLine(FrontService.service.ToString());
        }
    }
}

File: testresources/java/benchmark_2012/domain/foursquarealternative/kilroy/Map.java
package domain.foursquarealternative.kilroy;

import domain.foursquarealternative.whrrl.FrontService;
//Functional requirement 3.2
//Test case 150: Class domain.foursquarealternative.kilroy.Map is not allowed to have a dependency with classes from domain.foursquarealternative.glympse
//Result: TRUE
public class Map {
	public Map(){
		System.out.println(FrontService.service.toString());
	}
}